pkg/kn/commands/service/export.go

```diff
@@ -60,9 +60,9 @@ func NewServiceExportCommand(p *commands.KnParams) *cobra.Command {
   # Export a service in JSON format
   kn service export foo -n bar -o json
   # Export a service with revisions
-  kn service export foo --with-revisions --mode=resources -n bar -o json
+  kn service export foo --with-revisions --mode=export -n bar -o json
   # Export services in kubectl friendly format, as a list kind, one service item for each revision
-  kn service export foo --with-revisions --mode=kubernetes -n bar -o json`,
+  kn service export foo --with-revisions --mode=replay -n bar -o json`,
```

Comment:
should we add a line separator in each example with this to align with other command examples?

Reply:
Sure, will do.

Reply:
PTAL https://github.com/knative/client/pull/1006/commits/4aaab55d8e7456dcae6809986a9d808f6bf2d87a.